tests/pages/remoteInstancesPage.js

```diff
@@ -179,6 +179,7 @@ module.exports = {
         I.click(this.fields.usePgStatStatements);
         break;
       case 'rds-mysql56':
```

Comment:
I assume the test that adds `RDS MySQL 5.6` instance is broken now

Reply:
It's not failing: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/8003/tests :)

Reply:
```suggestion
      case 'rds-mysql56' || 'pmm-qa-postgres-12':
```

Reply:
This solution doesn't work.